mobile/Screen/SubmissionScreen.js

```diff
@@ -1,20 +1,198 @@
+  const [platform, setPlatform] = useState('');
+  const [accountTag, setAccountTag] = useState('');
+  const [artworkTitle, setArtworkTitle] = useState('');
+  const [description, setDescription] = useState('');
+
+  const [images, setImages] = useState([]);
+
+  function clearInput() {
+    setName('');
+    setEmail('');
+    setPlatform('');
+    setAccountTag('');
+    setArtworkTitle('');
+    setDescription('');
+  }
+
+  const submit = async () => {
+    try {
+      const result = await axios.post(`${URL}/submissions/post`, {
+        name,
+        email,
+        socials: {
+          platform,
+          tag: accountTag,
+        },
+        title: artworkTitle,
+        description,
+      });
+      setSubmissions((prev) => ([...prev, result.data]));
+      clearInput();
+    } catch (err) {
+      console.error(err);
+    }
+  };
+
+  const pickImage = async () => {
+    const result = await ImagePicker.launchImageLibraryAsync({
+      mediaTypes: ImagePicker.MediaTypeOptions.All,
+      allowsEditing: true,
+      aspect: [4, 3],
+      quality: 1,
+    });
+
+    if (!result.canceled) {
+      setImages((prev) => ([...prev, result.assets[0].uri]));
+    }
+  };
+
+  const openCamera = async () => {
+    const permissionResult = await ImagePicker.requestCameraPermissionsAsync();
+
+    if (permissionResult.granted === false) {
+      return;
+    }
+
+    const result = await ImagePicker.launchCameraAsync();
+
+    if (!result.canceled) {
+      setImages((prev) => ([...prev, result.assets[0].uri]));
+    }
+  };
+
+  useEffect(() => {
+    (async () => {
+      const result = await axios.get(`${URL}/submissions/get`);
+      setSubmissions(result.data);
+    })();
+  }, []);
+
   return (
-    <View style={styles.container}>
-      <Text>Submission Screen</Text>
-    </View>
+    <ScrollView>
+      <View style={styles.container}>
+        <View>
+          <Button title="Image" onPress={pickImage} />
+          <Button title="Camera" onPress={openCamera} />
+          {images.map((src) => <Image source={{ uri: src }} style={{ width: 200, height: 200 }} />)}
+          <TextInput
+            styles={styles.input}
+            placeholder="Name"
+            onChangeText={(newName) => setName(newName)}
+            defaultValue={name}
+          />
+          <TextInput
+            styles={styles.input}
+            placeholder="Email"
+            onChangeText={(newEmail) => setEmail(newEmail)}
+            defaultValue={email}
+          />
+          <TextInput
+            styles={styles.input}
+            placeholder="Social Media Platform"
+            onChangeText={(newPlatform) => setPlatform(newPlatform)}
+            defaultValue={platform}
+          />
+          <TextInput
+            styles={styles.input}
+            placeholder="Social Media Account Tag"
+            onChangeText={(newEmail) => setAccountTag(newEmail)}
+            defaultValue={accountTag}
+          />
```

Comment:
This is fine here for now, but remember to remove later since you will likely be automatically adding a user's id instead to link the profile information